boot/assets.go

```diff
@@ -265,6 +265,9 @@ type TrustedAssetsInstallObserver struct {
 
 	dataEncryptionKey secboot.EncryptionKey
 	saveEncryptionKey secboot.EncryptionKey
+
+	// XXX: auxEncryptionKey will be "nil" for the TPM case currently
+	auxEncryptionKey *secboot.AuxKey
```

Comment:
The current equivalent for this in the TPM case is authKey created inside sealKeyToModeenvUsingSecboot in boot/seal.go, so it's a bit strange that the 2 keys are created in different places. I guess that is ok for now though - eventually the TPM implementation will use the same key as well

Reply:
Great point, given that right now this probably adds more confusion than clarity I'm inclined to close this PR and simply create the new auxKey inside the sealKeyToModeenvUsingFdeHook for symetry.